docs/ModStructure/index.md

```diff
@@ -0,0 +1,31 @@
+# Mod Structure
+
+Following the [Modding Basics](https://community.bistudio.com/wiki/DayZ:Modding_Basics), you need to create a `CfgMods` class entry in a config with the name of your mod. Make note of the name of the mod as this is really important, the following example is 'NameOfMod'.
+
+```csharp
+class CfgMods
+{
+	class NameOfMod
+	{
+	};
+};
+```
+
+In script, mod the `ModStructure` class and override `OnLoad` as follows. Replace 'NameOfMod' with the name you noted down earlier. Make sure to call super and return it's value if `modName` does not equal the name of your mod.
+
+Currently `CF_OnLoad` is only used for setting the mods current storage version for [ModStorage](../ModStorage/index.md) by calling `SetStorageVersion`.
+
+```csharp
+modded class ModStructure
+{
+	override bool CF_OnLoad(CF_String modName)
+	{
+		if (!CF_String.EqualsIgnoreCase(modName, "NameOfMod"))
+		{
+			return super.CF_OnLoad(modName);
+		}
+
+		return true;
+	}
+}
+```
```

Comment:
I don't quite understand this step? Why does the user have to do this? 

You mention `SetStorageVersion()` but I don't find a reference to it in the modstorage docs? It points back to this doc saying this is how to setup the version?

I thought we go through the cfgmods, read their name and their version from the game config and thats about it. Per default. If a modder maybe wants to rename his mod but keep an alias of it to access the old data, then i guess he could override something to say "ah yeah thats me" ... but this should not be required by "normal" users?

Reply:
> You mention SetStorageVersion() but I don't find a reference to it in the modstorage docs? It points back to this doc saying this is how to setup the version?

Must've accidentally deleted the line

> I thought we go through the cfgmods, read their name and their version from the game config and thats about it. Per default. If a modder maybe wants to rename his mod but keep an alias of it to access the old data, then i guess he could override something to say "ah yeah thats me" ... but this should not be required by "normal" users?

Forgot to document that, but yes that is the case. It is the `storageVersion` entry.